docs/concepts/governance/multisig.md

```diff
@@ -7,19 +7,18 @@ The core of Balancer smart contracts are immutable and do not use proxies or oth
+| Protocol Fees Multisig | Collect fees, and set A-Factors and Fees on pools (default pool-owner, except on mainnet where a separate Multisig is used to set fees. | [MAINNET](https://gnosis-safe.io/app/eth:0x7c68c42De679ffB0f16216154C996C354cF1161B/home), [ARBI](https://gnosis-safe.io/app/arb1:0x7c68c42De679ffB0f16216154C996C354cF1161B/home), [POLYGON](https://gnosis-safe.io/app/matic:0x7c68c42De679ffB0f16216154C996C354cF1161B/home) | `0x7c68c42De679ffB0f16216154C996C354cF1161B` | [BAL Maxis](#operational-multisigs-signer-set-aka-balancer-maxis)                                                                                                  |
+| Mainnet Fee Setter     | Default pool owner for Mainnet that can set A-Factors and protocol fees.                                                                | [MAINNET](https://gnosis-safe.io/app/eth:0xf4A80929163C5179Ca042E1B292F5EFBBE3D89e6/home)                                                                                                                                                                                       | `0xf4A80929163C5179Ca042E1B292F5EFBBE3D89e6` | [BAL Maxis](#operational-multisigs-signer-set-aka-balancer-maxis)                                                                                                  |
+| DAO Multlsig           | Funding BIPs, killing of gauges, veBAL whitelisting                                                                                     | [MAINNET](https://gnosis-safe.io/app/eth:0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f/home)                                                                                                                                                                                       | `0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f` | [DAO Signers](#dao-multisig-signer-set)                                                                                                                            |
+| LM Multisig            | Used to manage gauges and Reward Tokens and manage liquidity supplied to multichain (bridge). New Gauge requests go here.               | [MAINNET](https://gnosis-safe.io/app/eth:0xc38c5f97B34E175FFd35407fc91a937300E33860/home), [ARBI](https://gnosis-safe.io/app/arb1:0xc38c5f97B34E175FFd35407fc91a937300E33860/home), [POLYGON](https://gnosis-safe.io/app/matic:0xc38c5f97B34E175FFd35407fc91a937300E33860/home) | `0xc38c5f97B34E175FFd35407fc91a937300E33860` | [BAL Maxis](#operational-multisigs-signer-set-aka-balancer-maxis) + [Markus Koch - blabs](https://etherscan.io/address/0x6bB4720473d4D7133f944785e5EE1A650C07f34e) |
```

Comment:
@markusbkoch Your name is being connected to this signer address on the LM multisig here.  Just want to make sure to get sign-off from you before pushing that to the docs pages.